SnapshotTesting.xcodeproj/xcshareddata/xcschemes/SnapshotTesting_iOS.xcscheme

```diff
@@ -14,7 +14,7 @@
             buildForAnalyzing = "YES">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "NT_E1FAA066576312BD4B5DB23882D1278B"
+               BlueprintIdentifier = "64F17F6842573B100D61132C44E859BF"
```

Comment:
Do we wanna add XcodeGen as an SPM dependency everywhere?

Reply:
done https://github.com/pointfreeco/swift-snapshot-testing/pull/189/commits/3120fdfbed97cbeb0155594d075d9871e6347b62